### ISSUE TITLE
Luckydog read this

### ISSUE THREAD

**Issue by ArcidonYT** [luckydog7/Funkin-android#977]
why are there still black screen since you knew how to fix them? That happens in the kade engine and crashes when it loads ballistic and the dialoge is just black and red
![Screenshot_20210525_214025_com ninjamuffin99 funkin](https://user-images.githubusercontent.com/79063412/119551771-ac6b5900-bda2-11eb-9987-d9ef12697318.jpg)

![Screenshot_20210525_212756_com ninjamuffin99 funkin](https://user-images.githubusercontent.com/79063412/119551818-b8efb180-bda2-11eb-8c6b-df9865a76c71.jpg)

**Comment by ROBIKALIVE13** [comment]
> why are there still black screen since you knew how to fix them? That happens in the kade engine and crashes when it loads ballistic and the dialoge is just black and red
> ![Screenshot_20210525_214025_com ninjamuffin99 funkin](https://user-images.githubusercontent.com/79063412/119551771-ac6b5900-bda2-11eb-9987-d9ef12697318.jpg)
> 
> ![Screenshot_20210525_212756_com ninjamuffin99 funkin](https://user-images.githubusercontent.com/79063412/119551818-b8efb180-bda2-11eb-8c6b-df9865a76c71.jpg)
> 

it can be a ram problem

**Comment by Rob7686** [comment]
It is a ram proplem it works fine with me

**Comment by ArcidonYT** [comment]
I know

**Comment by Sprite-dev** [comment]
Welp Same With Me But I Dont Wanna waste Luckdog7

**Comment by Kris6528** [comment]
Maybe your phone is not supported on 64-bit and low ram

**Comment by Johnsen290** [comment]
that's because you have a phone that only has a OpenGL 2.0, new SDL cannot render because of that, try playing it on a newer system, what is you phone name?
It should be at least OpenGL 3.0 capable.

**Comment by ppllo** [comment]
THE RESOLUTION this is the problem

**Comment by ppllo** [comment]
your phone does not support the size and width of the characters' sprites

**Comment by ppllo** [comment]
Like mine

**Comment by Johnsen290** [comment]
what is your phone

**Comment by Johnsen290** [comment]
no, it can if you have at least an OpenGL 3.0 GPU capable phone.
this game can fit in any resolution, just like the PC version.

**Comment by Johnsen290** [comment]
the engine resizes the images, to fit in your screen.

**Comment by ArcidonYT** [comment]
> what is your phone

Honor 8S

**Comment by Johnsen290** [comment]
that actually should work, does the original port have black images?
also i might try the kade engine on my phone which is less better than your phone.

**Comment by ArcidonYT** [comment]
> that actually should work, does the original port have black images?
> also i might try the kade engine on my phone which is less better than your phone.

Doesnt work on all version except ver 5-6

**Comment by Johnsen290** [comment]
the latest version? that's fine because the old version had this issue even if it is capable but now fixed in the latest.

**Comment by Johnsen290** [comment]
Phone : Realme 5i
